src/MiniCover/Instrumentation/Instrumenter.cs

```diff
@@ -130,31 +130,11 @@ private void VisitAssemblyGroup(IEnumerable<string> assemblyFiles)
             File.Delete(instrumentedAssembly.TempPdbFile);
         }
 
-        private IAssemblyResolver GetAssemblyResolver(string assemblyFile)
+        private InstrumentedAssembly InstrumentAssemblyIfNecessary(string assemblyFile)
         {
-            var resolver = new CustomAssemblyResolver();
-
             var assemblyDirectory = Path.GetDirectoryName(assemblyFile);
-            resolver.AddSearchDirectory(assemblyDirectory);
-
-            var additionalPaths = new List<string>();
-            var runtimeConfigPaths = FileUtils.GetFiles(null, null, "**/*.runtimeconfig.dev.json", assemblyDirectory);
```

Comment:
We can use Directory.GetFiles instead of using glob patterns. It should be faster and doesn't require additional libraries for glob processing.